prow/bugzilla/types.go

```diff
@@ -226,6 +226,7 @@ type Flag struct {
 type BugUpdate struct {
 	// DependsOn specifies the bugs that this bug depends on
 	DependsOn  *IDUpdate `json:"depends_on,omitempty"`
+	Blocks     *IDUpdate `json:"blocks,omitempty"`
```

Comment:
Of note: even though this is the inverse of the `DependsOn` field, they are entirely unrelated in the API.